lib/datadog/security/contrib/rack/patcher.rb

```diff
@@ -0,0 +1,99 @@
+require 'datadog/security/contrib/patcher'
+require 'datadog/security/contrib/rack/integration'
+
+module Datadog
+  module Security
+    module Contrib
+      module Rack
+        module Patcher
+          include Datadog::Security::Contrib::Patcher
+
+          module_function
+
+          def patched?
+            Patcher.instance_variable_get(:@patched)
+          end
+
+          def target_version
+            Integration.version
+          end
+
+          # TODO: logger
```

Comment:
I was gonna say that this probably can't stay like this in prod, without a proper logger configuration, but then I saw the `TODO` 😄.

Reply:
Yes I did not bother to do a proper logger as I thing this facility should move to the core component.